packages/documentation/src/stories/utilities/float/float.snapshot.stories.ts

```diff
@@ -0,0 +1,46 @@
+import type { Args, StoryContext, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import meta from './float.stories';
+import { bombArgs } from '@/utils';
+
+const { id, ...metaWithoutId } = meta;
+
+export default {
+  ...metaWithoutId,
+  title: 'Snapshots',
+};
+
+type Story = StoryObj;
+
+export const Float: Story = {
+  render: (_args: Args, context: StoryContext) => {
+    return html`
+      <div class="snapshot-example">
+        <h1>Snapshot with Float</h1>
+        ${['bg-white', 'bg-dark'].map(
```

Comment:
Testing a floating utility on different colored backgrounds, does not really bring any advantage. But since I told you to do so, I'm ok if you keep it this way 😉